<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormConsole</class>
 <widget class="QWidget" name="FormConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pb_file">
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_add">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_start">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="icon">
         <iconset resource="../qiperf.qrc">
          <normaloff>:/start</normaloff>:/start</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_stop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="icon">
         <iconset resource="../qiperf.qrc">
          <normaloff>:/stop</normaloff>:/stop</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string notr="true">Group</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Pair Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Run Status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>client</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>server</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string> Average(Mbps)</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="w_chart" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="vl_chart">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qiperf.qrc"/>
 </resources>
 <connections/>
</ui>
